libs/sdk-bindings/bindings-swift/Sources/BreezSDK/TaskProtocol.swift

```diff
@@ -19,6 +19,10 @@ extension TaskProtocol {
             return
         }
         
+        if body != nil {
+            bestAttemptContent.body = body!
+        }
+
         if threadIdentifier != nil {
             bestAttemptContent.threadIdentifier = threadIdentifier!
         }
```

Comment:
```suggestion
        if let body = body {
            bestAttemptContent.body = body
        }

        if let threadIdentifier = threadIdentifier {
            bestAttemptContent.threadIdentifier = threadIdentifier
        }
```
I think it's safer to use optional binding here instead of force unwrapping.

Reply:
Thanks @erdemyerebasmaz !